### PR TITLE
Add Datalab and C2D Bioconductor documentation.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: docs/source/includes/bioconductor_deployment_sidebar.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.. sidebar:: Details
+
+  This will create a virtual machine on Google Cloud Platform with a locked down network (only SSH port 22 open).  Your local machine will securely connect to the VM via an ssh tunnel.
+
+  It will also mount an persistent disk underneath ``/home/rstudio/data``.  Store important files there.  Docker containers are stateless, so if the container restarts for any reason, and files you created within the container will be lost.
```
Comment (mbookman) [inline docs/source/includes/bioconductor_deployment_sidebar.rst]: s/an/a
s/underneath/under
s/and files/then files

File: docs/source/use_cases/run_familiar_tools/bioconductor.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+.. include:: /includes/bioconductor_deployment_sidebar.rst
+
+Bioconductor maintains Docker containers with R, Bioconductor packages, and RStudio Server all ready to go!  Its a great way to set up your R environment quickly and start working.  The instructions to deploy it to Google Compute Engine are below but if you want to learn more about these containers, see http://www.bioconductor.org/help/docker/.
+
+1. Click on `click-to-deploy Bioconductor`_ to navigate to the deployer page on the Developers Console.
```
Comment (mbookman) [inline docs/source/use_cases/run_familiar_tools/bioconductor.rst]: s/deployer/launcher ?

File: docs/source/use_cases/run_familiar_tools/bioconductor.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  2. Optional: change the *Machine type* if you would like to deploy a machine with more RAM.
```
Comment (mbookman) [inline docs/source/use_cases/run_familiar_tools/bioconductor.rst]: "with more CPU cores or RAM"

**Comment by deflaux** [comment]
@mbookman thanks for the feedback!  PTAL

I went ahead and added these to the current TOC but please merge these to staging-2 and add to your new TOC structure.

File: docs/source/use_cases/run_familiar_tools/bioconductor.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  3. Optional: change the *Data disk size (GB)* if you would like to use a larger persistent disk for your own files.
+  4. Optional: change *Docker image* if you would like to install a container with additional Bioconductor packages preinstalled.
+5. Click on the *Deploy Bioconductor* button.
+6. Follow the post-deployment instructions to log into RStudioServer via your browser!
+
+If you want to deploy a different docker container, such as the one from :doc:`/workshops/bioc-2015`
+
+1. In field *Docker Image* choose item ``custom``.
+2. Click on *More* to display the additional form fields.
+3. In field *Custom docker image* paste in value ``gcr.io/bioc_2015/devel_sequencing``.
```
Comment (mbookman) [inline docs/source/use_cases/run_familiar_tools/bioconductor.rst]: Should probably say:

 3. In field *Custom docker image* paste in the docker image path, such as ``gcr.io/bioc_2015/devel_sequencing``

Comment (deflaux) [inline docs/source/use_cases/run_familiar_tools/bioconductor.rst]: Agreed, good catch.

Also (note to self) I need to be more careful of my use of "container" versus "image" http://stackoverflow.com/questions/23735149/docker-image-vs-container

**Comment by deflaux** [comment]
This change mostly went in with https://github.com/googlegenomics/readthedocs/pull/82

**Pull request merged**